AutoCollection/Statsbeat.ts

```diff
@@ -232,17 +275,23 @@ class Statsbeat {
     private _getResourceProvider(callback: () => void) {
         // Check resource provider
         let waiting: boolean = false;
+        this._resourceIdentifier = Constants.StatsbeatResourceProvider.unknown;
         if (process.env.WEBSITE_SITE_NAME) { // Web apps
             this._resourceProvider = Constants.StatsbeatResourceProvider.appsvc;
+            this._resourceIdentifier = process.env.WEBSITE_SITE_NAME;
         } else if (process.env.FUNCTIONS_WORKER_RUNTIME) { // Function apps
             this._resourceProvider = Constants.StatsbeatResourceProvider.function;
+            if (process.env.WEBSITE_HOSTNAME) {
+                this._resourceIdentifier = process.env.WEBSITE_HOSTNAME;
+            }
         } else if (this._config) {
```

Comment:
What if the application is not in any of these rps? We probably shouldn't be sending this stats at all.

Reply:
rp has a default case: "unknown".  like standalone apps.. it's not an attach scenario.

Reply:
Right, so we should not be sending any of the "attach" metrics in this default case.

Reply:
@heyams can you confirm we don't send attach Statbeat for unknown

Reply:
I would like to track unknown as well.. i.e. non-attach vs. attach.  that's a good data trend to show.

Reply:
What do we populate for properties then? Do we just leave empty?